### PR TITLE
 Update the administrator documentation pages

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: doc/administrator/editing.rst [hunk 1]
```diff
@@ -3,12 +3,10 @@
...
 =======
 
-This section describes how to set up feature editing in c2cgeoportal
-applications.
+This section explains how to configure the edit function in c2cgeoportal.
 
-Just like most administrative tasks setting up editing in a c2cgeoportal
-application involves intervening in the database, through the c2cgeoportal
-administration interface.
+Just like most administrative tasks the configuration of the edition in the c2cgeoportal application
```
Comment (jwkaltz) [inline doc/administrator/editing.rst]: Suggestion: "Just like most admininistrative tasks, the configuration of the editing functionality involves intervening in the database via the c2cgeoportal administration interface."

File: doc/administrator/mapfile.rst [hunk 2]
```diff
@@ -8,29 +8,20 @@ administrator manages the application through the database, and the
...
 
 The application's mapfile is located in the ``mapserver`` directory, it is
 commonly named ``mapserver.map.mako``.
 
 .. note::
 
     The application's mapfile is a ``.mako`` file because it contains variables.
-    These variables are substituted when the ``make`` build command is
-    executed.
+    These variables are substituted when the ``make`` build command is executed.
```
Comment (jwkaltz) [inline doc/administrator/mapfile.rst]: is this uptodate regarding docker world?

File: doc/administrator/mapfile.rst [hunk 3]
```diff
@@ -61,16 +55,18 @@ resolution into account.
 WFS GetFeature
 --------------
 
-Layers involved in the ``box query`` and ``query builder`` features should
-support WFS GetFeature. To support WFS GetFeature a ``LAYER`` should define
-a template::
+Layers involved in the ``query`` (box and point), the ``filters``, and the WFS permalink features should
```
Comment (jwkaltz) [inline doc/administrator/mapfile.rst]: Are we saying:
"Layers that you want to use in a ``query`` (box and point), in ``filters``, or for WFS permalink features must support WFS GetFeature."
?

File: doc/administrator/mapfile.rst [hunk 4]
```diff
@@ -160,30 +153,29 @@ example::
...
-and vice versa.
+restricted in the mapfile should be defined as well in the admin interface and vice versa.
 
 With a RestrictionArea area
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 A RestrictionArea is used to restricted the layer displaying to a given area.
-This area is specified in the administration interface while defining the
-``RestrictionArea`` element.
+This area is specified in the administration interface while defining the ``RestrictionArea`` element.
```
Comment (jwkaltz) [inline doc/administrator/mapfile.rst]: "while defining" -> "by defining"

File: doc/administrator/tinyows.rst [hunk 5]
```diff
@@ -2,46 +2,24 @@
 
 WFS-T with TinyOWS
 ==================
-Based on `TinyOWS <http://mapserver.org/tinyows/>`__ c2cgeoportal layers can be
+
+Based on `TinyOWS <http://mapserver.org/tinyows/>`__ c2cgeoportal mapserver layers can be
```
Comment (jwkaltz) [inline doc/administrator/tinyows.rst]: Why mapserver layers, it also for qgisserver layers AFAIK?

File: doc/administrator/mapfile.rst (hunk 2 above)
Comment (sbrunner) [reply doc/administrator/mapfile.rst]: Yes it steel valid :-)

File: doc/administrator/mapfile.rst (hunk 3 above)
Comment (sbrunner) [reply doc/administrator/mapfile.rst]: Yes, will be updated :-)

File: doc/administrator/tinyows.rst (hunk 5 above)
Comment (sbrunner) [reply doc/administrator/tinyows.rst]: It can be but QGIS server directly support WFS-T, I will add a note about that :-)

**Comment by sbrunner** [comment]
Updated :-)

**Pull request merged**